Tiltfile

```diff
@@ -18,10 +18,9 @@ helm_resource(
     image_deps=['timberio/vector'],
     image_keys=[('image.repository', 'image.tag')],
     flags=[
-        '--devel',
         '--set', 'role=Agent',
-        # '--set', 'env[0].name=VECTOR_LOG',
-        # '--set', 'env[0].value=trace'
+        '--set', 'env[0].name=VECTOR_LOG',
+        '--set', 'env[0].value=trace'
```

Comment:
Is this setting only for debugging purposes? The deployed collector should be at WARN level, as even INFO is too chatty to easily see problems.